docs/guides/websites/hosting/secure-website-lets-encrypt-acme-sh/index.md

```diff
@@ -1,12 +1,14 @@
 ---
+slug: secure-website-lets-encrypt-acme-sh
```

Comment:
Our new docs UI requires us to set a slug in each guide, and the slug is generally equal to the name of the filesystem directory that is created for the guide. 

If you use Hugo archetypes when creating any new guides in the future, our archetype will automatically set this for you:
https://github.com/linode/docs/blob/develop/CONTRIBUTING.md#create-a-new-guide